x/docs/md/cli/linux.md

```diff
@@ -34,7 +34,7 @@ To check which shell you have, run `echo $SHELL`.
 
 These are not officially supported, and sometimes (often) lag behind the latest release.
 
-There's a [freshports build](http://www.freshports.org/misc/exercism), an [AUR package](https://aur.archlinux.org/packages/exercism-cli), and an [openSUSE package](https://software.opensuse.org/package/golang-github-exercism-cli).
+There's a [freshports build](http://www.freshports.org/misc/exercism), an [AUR package](https://aur.archlinux.org/packages/exercism-cli), an [openSUSE package](https://software.opensuse.org/package/golang-github-exercism-cli) and a [Nix/NixOS package](https://github.com/NixOS/nixpkgs/blob/827d48e6b383a47185c546fd1bf69296a92b02b8/pkgs/applications/misc/exercism/default.nix).
```

Comment:
Is there a reason why you've linked to this specific version rather than tracking `master`?

Reply:
@Insti actually it was a bad copy paste. Thanks for the sharp eye! Will fix it soon